xml/System.Configuration/KeyValueConfigurationCollection.xml

```diff
@@ -373,7 +373,7 @@
       </Parameters>
       <Docs>
         <param name="key">A string specifying the <c>key</c>.</param>
-        <summary>Removes a <see cref="T:System.Configuration.KeyValueConfigurationElement" /> object from the collection.</summary>
+        <summary>Removes a <see cref="T:System.Configuration.KeyValueConfigurationElement" /> object from the collection. It will not throw an exception if the item is not found.</summary>
```

Comment:
The additional sentence is better suited for the Remarks section than the method description.